.readthedocs.yml

```
@@ -1,15 +1,42 @@
+# version: 2
+
+# sphinx:
+#   configuration: doc/conf.py
+
+# python:
+#     version: "3.9"
+#     install:
+#         - requirements: doc/requirements.txt
+#         - method: pip
+#           path: .
+#     system_packages: true
+
+# build:
+#     image: latest
+#     tools:
+#         python: "3.9"
+
+# Required
 version: 2
 
+# Set the version of Python and other tools you might need
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.9"
+
+# Build documentation in the docs/ directory with Sphinx
 sphinx:
-  configuration: doc/conf.py
+   configuration: doc/conf.py
 
-python:
-    version: 3.8
-    install:
-        - requirements: doc/requirements.txt
-        - method: pip
-          path: .
-    system_packages: true
+# If using Sphinx, optionally build your docs in additional formats such as PDF
+# formats:
+#    - pdf
 
-build:
-    image: latest
+# Optionally declare the Python requirements required to build your docs
+python:
+   install:
+   - requirements: doc/requirements.txt
+   - method: pip
+     path: .
+   system_packages: true
```

missing new line at the end of file
```suggestion
   system_packages: true

```